tests/ColorTests/FunctionalTest/color_ft.cpp

```diff
@@ -161,6 +165,28 @@ TEST_P(color_functional_test, color_streaming_test)
             break;
         }
 
+        if (ts_init)
+        {
+            // Ensure the device and system time stamps are increasing, image might get dropped, which is ok for this
+            // portion of the test.
+            uint64_t ts;
+            ts = k4a_image_get_device_timestamp_usec(image);
+            ASSERT_GT(ts, ts_d); // We don't test upper max, because we expect the next sample to be larger, this also
```

Comment:
Since we don't initially set `ts_d` to a value I'd be very impressed if all of our compilers pass by this without complaining about a possible use before assignment.